fundamentals/fundamentals-collections/src/main/scala/izumi/functional/IzEitherAggregations.scala

```diff
@@ -0,0 +1,83 @@
+package izumi.functional
+
+import izumi.functional.IzEitherAggregations.{EitherBiAggregate, EitherBiFlatAggregate, EitherBiFlatMapAggregate, EitherBiMapAggregate}
+import izumi.fundamentals.collections.nonempty.NonEmptyList
+
+import scala.collection.compat.*
+import scala.language.implicitConversions
+
+trait IzEitherAggregations extends IzEitherTraversals {
+  @inline implicit final def EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], R]]
+  ): EitherBiAggregate[L, R, Col] = new EitherBiAggregate(col)
+  @inline implicit final def EitherBiFlatAggregate[L, R, Col[x] <: IterableOnce[x], Col2[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], Col2[R]]]
+  ): EitherBiFlatAggregate[L, R, Col, Col2] = new EitherBiFlatAggregate(col)
+  @inline implicit final def EitherBiMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiMapAggregate[Col, T] = new EitherBiMapAggregate(col)
+  @inline implicit final def EitherBiFlatMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFlatMapAggregate[Col, T] = new EitherBiFlatMapAggregate(col)
+}
+
+object IzEitherAggregations extends IzEitherAggregations {
```

Comment:
One method is still missing a NonEmptyList version - `biAggregateScalar` (you may also split `lrPartition` and put it into IzEitherTraversals - since it does not depend on List)